products/org.aposin.gem.product/org.aposin.gem.ui.product

```diff
@@ -10,7 +10,7 @@
       <vmArgs>-Xmx75M
 -Dswt.autoScale=true
 -Dorg.aposin.gem.gitconfig.url=git@github.com:aposin/gem-config.git
--Dorg.aposin.gem.gitconfig.branch=v0.3.0
+-Dorg.aposin.gem.gitconfig.branch=main
```

Comment:
Is this "main" correct?

Reply:
Yes, this is the development branch for the config. In the README you can see how is the process for that (https://github.com/aposin/gem#releasing-a-new-version)